examples/hosts/app-integration/external-data/src/mock-external-data-service/service.ts

```diff
@@ -0,0 +1,230 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Server } from "http";
+
+import cors from "cors";
+import express from "express";
+import { isWebUri } from "valid-url";
+
+import { assertValidTaskData, TaskData } from "../model-interface";
+import { MockWebhook } from "../utilities";
+import { ExternalDataSource } from "./externalDataSource";
+
+/**
+ * {@link initializeExternalDataService} input properties.
+ */
+export interface ServiceProps {
+	/**
+	 * Port to listen on.
+	 */
+	port: number | string;
+
+	/**
+	 * External data source backing this service.
+	 *
+	 * @defaultValue A new data source will be initialized.
+	 */
+	externalDataSource?: ExternalDataSource;
+}
+
+/**
+ * Initializes the mock external data service.
+ *
+ * @remarks Consumers are required to manually dispose of the returned `Server` object.
```

Comment:
We can probably remove these remarks if things are not IDisposable anymore.

Reply:
🤦‍♂️